src/backend/joanie/core/serializers.py

```diff
@@ -133,7 +370,6 @@ class Meta:
         read_only_fields = [
             "created_on",
             "state",
-            "price",
```

Comment:
I find it weird that this list if not exhaustive. We should add a test to secure that this field is indeed readonly if we don't have it already

Reply:
I thought that was redundant as we declare explicitly price as `read_only` just above. But in order to improve readability, I can add all read only fields within this list and apply this accross all serializers.

Elsewhere a test exists to check that order serializer read only fields are not writable.